test-cases/gutenberg/color-settings.md

```diff
@@ -0,0 +1,183 @@
+
+# Color Settings - Test Cases
+
+#### **Precondition**
+
+Added block with color support such as `Buttons` or `Cover` 
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### Colors segmented controls
+
+**Steps:**
+
+* Press `Background Color` / `Color`
+* Switch several times between `Solid` and `Gradient` segments 
+
+**Expected behavior:**
+
+* Selection animation works smoothly
+* Color swatches fade parallel with active segment change
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Scroll color palette to start
+
+**Steps:**
+
+* Press `Background Color` / `Color`
+* Select one of the last solid colors (_blue or purple_)
+* Select `Gradient` segment
+
+**Expected behavior:**
+
+* Color list is scrolled to the start
+* Color swatches fade parallel with scroll
```

Comment:
Again a gif would really help, I don't know what to expect by just reading

Reply:
done!